python/src/GNDStk.python.cpp

```diff
@@ -12,6 +12,7 @@ namespace core {
 
   void wrapNode( python::module& );
 
+  void wrapGridStyle( python::module& );
```

Comment:
And I know there are some other Standard Interface related enumerators floating around.

Reply:
I was thinking of adding all of them when I added this one but I'll do that once I'm done with all the other prototype testing. There was one in there (the encoding one) that I was wondering about what it was for.